com.github.ztefn.haguichi.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.github.ztefn.haguichi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "haguichi",
+    "finish-args": [
+        "--filesystem=/var/lib/logmein-hamachi",
+
+        "--system-talk-name=org.freedesktop.login1",
+
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.freedesktop.Notifications",
+
+        "--share=network",
+        "--share=ipc",
+
+        "--socket=x11",
```

Comment:
If Haguichi runs on Wayland, you can use fallback-x11 here.

Reply:
I had no idea, thanks!

Reply:
Hmm I got this after logging in on X the second time:

```
Unable to init server: Could not connect: Connection refused

(haguichi:2): Gtk-WARNING **: 19:56:58.440: cannot open display:
```

Reply:
@p1u3o are you sure this fallback-x11 is prime time ready?

Reply:
I've been using it a few manifests, it disables the X socket under Wayland irrc.

Reply:
The display issue when using fallback-x11 seems to occur only when switching session from X11 to Wayland and back to X11. I suppose not too big of a deal.